content/blog/2018/beyondtellerand.md

```diff
@@ -45,7 +45,7 @@ lesson on ethics in tech. How could I even begin to match the
 inspirational scope of these other presenters, while talking about
 layout?
 
-I couldn't --but I did have some fun. Inspired by the generated art from
+I couldn't -- but I did have some fun. Inspired by the generated art from
```

Comment:
I went down a rabbit hole... how did we end up with so many of these throughout?

Reply:
pandoc isn't great.